AmericanMedicalAssociation/AMA-style-guide
# Ama 41 promo block summary atom

Note: Please review and merge https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/12 first, then https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/13 second as this branch is based on those.

https://palantir.atlassian.net/browse/AMA-41

Adds the summary atom to the Promotional block.

### Steps to Test:

1. Navigate to Molecules > Blocks > Promotional Block.
2. Observe that there are three Promotional Blocks with primary ctas, titles, and summaries.
